src/core/ext/transport/chaotic_good/client_transport.cc

```diff
@@ -110,12 +112,16 @@ auto ChaoticGoodClientTransport::PushFrameIntoCall(ServerFragmentFrame frame,
             },
             []() -> StatusFlag { return Success{}; });
       });
+  // Wrap the actual sequence with something that owns the call handler so that
+  // its lifetime extends until the push completes.
+  return [call_handler, push = std::move(push)]() mutable { return push(); };
```

Comment:
this is the equivalent of `call_handler=call_handler`, so it introduces a new member in the lambda that just happens to have the same name, and yeah it's to lifetime extend because push() needs call_handler to be alive